livekit-android-sdk/src/main/java/io/livekit/android/e2ee/E2EEManager.kt

```diff
@@ -122,6 +137,21 @@ constructor(keyProvider: KeyProvider) {
         }
     }
 
+    public fun removePublishedTrack(track: Track, publication: TrackPublication, participant: LocalParticipant, room: Room) {
+        for (item in frameCryptors.entries) {
+            var key = item.key
+            var trackId = publication.sid
+            var participantId = participant.identity
+            if (key.first == trackId && key.second == participantId) {
```

Comment:
If we're checking against both parts of the pair, can just do `frameCryptors[trackId to participantId]` to grab the frameCryptor if any.

Reply:
How does this look like, https://github.com/livekit/client-sdk-android/pull/276/commits/d9fc42e0fc7a6654fe0ddb52f4097f3ae2eb71e4